src/utils.coffee

```diff
@@ -86,7 +86,7 @@ process_if = (ifValue)->
 # Parse layer options
 # @return [string] layer transformation string
 # @private
-process_layer = (layer)->
+process_layer = (layer, escapeQuotes = true)->
```

Comment:
Can you suggest a better param name than `escapeQuotes`?
As I see, it's about comma and slash, not about quotes.

Reply:
@shedar , how about "shouldEscape"?